subcommands/targets/offline-update.go

```diff
@@ -78,7 +78,12 @@ func doOfflineUpdate(cmd *cobra.Command, args []string) {
 	subcommands.DieNotNil(downloadOstree(factory, ti.ostreeVersion, ti.hardwareID, dstDir), "Failed to download Target's ostree repo:")
 
 	fmt.Printf("Downloading Apps fetched by the `assemble-system-image` run; build number:  %d, tag: %s...\n", ti.version, ti.buildTag)
-	subcommands.DieNotNil(downloadApps(factory, targetName, ti.version, ti.buildTag, path.Join(dstDir, "apps")), "Failed to download Target's Apps:")
+	err = downloadApps(factory, targetName, ti.version, ti.buildTag, path.Join(dstDir, "apps"))
+	if herr := client.AsHttpError(err); herr != nil && herr.Response.StatusCode == 404 {
+		fmt.Println("Apps of the specified Target has not been found, the update won't include Apps!!!")
```

Comment:
Maybe `WARNING: This Target does not include any Apps`

and once you go passed draft, you may add a little more context into the commit message

Reply:
>This Target does not include any Apps

But Target includes Apps, the `assemble` with fetched Apps is missing.

I see two use-cases:

1. There are no Apps in Target indeed
2. There is no `assemble` run with the fetched Apps artifact.

We can detect which of the use cases it is in the business logic of the command.  Or, print a warning message saying that this is either #1 or #2.